sessionctx/variable/varsutil.go

```diff
@@ -339,6 +343,30 @@ func TiDBOptMultiStmt(opt string) int {
 	return WarnInt
 }
 
+// ClusteredIndexDefMode controls the default clustered property for primary key.
+type ClusteredIndexDefMode int
+
+const (
+	// IntOnlyClustered indicates only single int primary key will default be clustered.
+	IntOnlyClustered ClusteredIndexDefMode = 0
```

Comment:
I suggest to name it `ClusteredIndexDefModeIntOnly`, `ClusteredIndexDefModeOff`, `ClusteredIndexDefModeOn` for more friendly autocomplete.

Reply:
renamed~